# Parameter Configuration File for MPC Trajectory Generator

# ROS Topic Names
pose_topic: "gt_pose"
scan_topic: "scan"
drive_topic: "nav"

# Visualization
visualization_enabled: true
obstacle_avoidance: true

# Input CSV File Path
csv_path: "/home/yash/yasht_ws/src/mpc_av/sensor_data/gtpose_fast.csv"
csv_trajectories_path: "/home/yash/yasht_ws/src/mpc_av/trajectories.csv"

# Car Parameters:
# Speed Variations
high_speed: 3.5
medium_speed: 2.5
low_speed: 2.0

# Tracking Parameter
lookahead_distance: 1.6

# WayPoint Input Method
# Options- 'user' - Multiple Inputs using interactive markers
#          'raceline' - PreComputed Raceline WayPoints
waypoint_input: "user"

# Local Map Parameter
local_map_length: 1.6
inflation_radius: 4 #cells